### ISSUE TITLE
consider breaking apart var calling from WholeGenome pipelines

### ISSUE THREAD

**Issue by SHuang-Broad** [broadinstitute/long-read-pipelines#293]
The whole genome pipelines now contains two main functional domains: 

1) merging alignments and QC
2) variant calling

Given that 2) is already pretty complex and 1) is going to become more complex, and we could have a 3) later (variant QC), we may need to decompose the WholeGenome pipelines.

At minimum, we need to make the main functional areas self-contained.

**Comment by SHuang-Broad** [comment]
#302 paves the way for this

**Comment by kvg** [comment]
Yeah, given that we're getting better and better at running automated analyses, separating these functionalities into two separate workflows seems quite sensible to me.
